### PR TITLE
Feature/input validation

### PR DESCRIPTION
**Pull request by j-or**
Includes:
- Validating and disabling data points (upper boundary, lower boundary, duplicate names/ids...)
- Allowing "," and "." in input model and data points
- Editing input params (for now still disabled when data points exist)

### CODE REVIEW

File: packages/ui/src/features/input-model/variable-editor.tsx [hunk 1]
```diff
@@ -1,26 +1,64 @@
...
+      editingCategoricalVariable === undefined,
+    [editingValueVariable, editingCategoricalVariable]
+  )
+
+  const [tabIndex, setTabIndex] = useState<number>(isValueTabSelected ? 0 : 1)
 
-  const [tabIndex, setTabIndex] = useState(0)
   const { classes } = useStyles()
 
+  useEffect(() => {
```
Comment (langdal) [inline packages/ui/src/features/input-model/variable-editor.tsx:58]: useEffect reminder

File: sample-app/index.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="icon" href="/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Process Optimizer Frontend</title>
+    <title>Brownie Bee</title>
```
Comment (langdal) [inline sample-app/index.html:7]: Maybe stick to Process Optimizer Frontend

**Review by langdal — APPROVED** [review]

File: sample-app/index.html (hunk 2 above)
Comment (j-or) [reply sample-app/index.html]: Changed

File: packages/ui/src/features/input-model/variable-editor.tsx (hunk 1 above)
Comment (j-or) [reply packages/ui/src/features/input-model/variable-editor.tsx:58]: Kept as is

File: packages/ui/src/features/input-model/categorical-variable.tsx [hunk 3]
```diff
@@ -47,11 +73,40 @@ export default function CategoricalVariable(props: CategoricalVariableProps) {
     }
   }, [formState, reset, isOptionsValid])
 
+  useEffect(() => {
```
[the remark this answers is not in the document]
Comment (j-or) [reply packages/ui/src/features/input-model/categorical-variable.tsx:76]: Kept as is

**Pull request merged**